src/components/Allocation/AddEditAllocationModal.tsx

```diff
@@ -64,6 +68,27 @@ export const AddEditAllocationModal = ({
     'walletAddress' | 'juiceboxProject'
   >('walletAddress')
 
+  const ethPaymentTerminalFee = useETHPaymentTerminalFee()
+  const amount = Form.useWatch('amount', form)
+
+  const amountAfterFee = useMemo(() => {
+    if (!ethPaymentTerminalFee || !amount?.amount) return
+    const percentageIncludedAfterFee =
+      Math.abs(100 - parseFloat(formatFee(ethPaymentTerminalFee))) / 100
+    const amountLeftover =
+      parseFloat(amount.amount) * percentageIncludedAfterFee
+    return formatCurrencyAmount({
+      amount: amountLeftover,
+      currency: allocationCurrency,
+    })
+  }, [allocationCurrency, amount?.amount, ethPaymentTerminalFee])
```

Comment:
I think we have a helper fn for this already: https://github.com/jbx-protocol/juice-interface/blob/c27300806df1409d7a140ca5e108bf16624626f3/src/utils/v2v3/math.ts#L247-L254